docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -215,7 +216,9 @@ include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 
 ====
 
-NOTE: For restore to be successful you have to give restore configuration at the `Hazelcast` custom resource creation. Otherwise, triggering restore on currently running members will likely to fail. 
+NOTE: For restore to be successful you have to give restore configuration at the `Hazelcast` custom resource creation. Otherwise, triggering restore on currently running members will likely to fail.
```

Comment:
```suggestion
[NOTE]
====
. You must configure restore at the creation of the `Hazelcast` custom resource. If this is not done, triggering restore on currently running members is likely to fail.
. You cannot override existing restore data using a backup. This means that, if a `PersistentValue` is left from the previousHazelcast cluster, you cannot use a backup to restore it.
====
```

Reply:
I see this is used several times, but I'm not sure if I have correctly understood; just let me know if I haven't and we can work together to make sure the right message is there.